src/allmydata/util/base32.py

```diff
@@ -144,16 +94,16 @@ def _b2a_l(os, lengthinbits):
 
 NUM_QS_TO_NUM_OS=(0, 1, 1, 2, 2, 3, 3, 4)
 NUM_QS_LEGIT=(1, 0, 1, 0, 1, 1, 0, 1,)
-NUM_QS_TO_NUM_BITS=tuple(map(lambda x: x*8, NUM_QS_TO_NUM_OS))
+NUM_QS_TO_NUM_BITS=tuple([x*8 for x in NUM_QS_TO_NUM_OS])
```

Comment:
This leaks `x` into the module namespace.  Not sure it really matters.  Might be worth fixing if any other changes are required.